modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -185,6 +185,92 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "Verify it's possible to add an object entry with the trigger On Order Status Update"
+	@priority = "4"
+	test CanCreateActionAddObjectEntryWithOnOrderStatusUpdate {
+		task ("Given an action in the Commerce Order object") {
```

Comment:
Use "Given an action in the Commerce Order triggered with Custom Object" because exemplifies better what is given to.